Cloud changelog - Aug 25th (#5847)
mattermost/docs: source/install/cloud-changelog.md

Latest Mattermost Cloud releases:

- [Release 2022-08-25](#release-2022-08-25)
- [Release 2022-08-18](#release-2022-08-18)
- [Release 2022-08-10](#release-2022-08-10)
- [Release 2022-08-03](#release-2022-08-03)
- [Release 2022-07-28](#release-2022-07-28)
- [Release 2022-07-20](#release-2022-07-20)

## Release 2022-08-25

### Highlights

### Improvements

#### User Interface (UI)

>  - Added the "Save" option to the post menu.

Reviewer: ```suggestion
 - Added the **Save** option to the post menu.
```

 - Added a banner to communicate delinquency to Cloud Customers.
 - Cloud instances with a file limit (Starter and Professional subscriptions) now display icons notifying that the limit has been hit and that archived files are unavailable until upgrade.

#### Administration
 - Started tracking the join time of team members and added a new API endpoint to retrieve information about team members who have joined during a given time.
 - Introduced an optional ``shouldRender`` function parameter to ``registerchannelHeaderMenuAction`` plugin function. This allows menu items to conditionally render depending on the current state prior to rendering.

### Bug Fixes
 - Fixed an issue where exports generated via mmctl without attachments still included the file properties in the post, so they couldn't be imported.
 - Fixed an issue that caused a crash when unread posts were fetched.

### Known Issues

>  - Mentions incorrectly shows users as not in a channel [MM-44157](https://mattermost.atlassian.net/browse/MM-44157).

Reviewer: ```suggestion
 - Mentions incorrectly show users as not in a channel [MM-44157](https://mattermost.atlassian.net/browse/MM-44157).
```

 - System Roles shows **License** and **Environment** as possible permissions, but they are always hidden in Cloud.
 - The Playbooks left-hand sidebar does not update when a user is added to a run or playbook without a refresh.
 - The runs and playbooks in the Playbooks left-hand sidebar does not have dot-menus that allow interaction with each item [MM-44752](https://mattermost.atlassian.net/browse/MM-44752).

>  - On the new Boards RHS from the channel Apps Bar, channel members who are not admins of the board are incorrectly able to see the "unlink" board button. However, clicking on the button will not actually unlink the board unless the user is a board admin [issue-focalboard-3600](https://github.com/mattermost/focalboard/issues/3600).

Reviewer: ```suggestion
 - On the new Boards RHS from the channel Apps Bar, channel members who are not board admins are incorrectly able to see the **Unlink** board button. However, selecting the button doesn't actually unlink the board unless the user is a board admin [issue-focalboard-3600](https://github.com/mattermost/focalboard/issues/3600).
```

>  - On Boards, clicking on `+ New` button below a column on the Kanban view does not always create a new card. As a workaround, set a new default card template by going to the dropdown menu from the blue `New` button on the header of the board, then open the Options Menu on any card template and select "Set as default" [issue-focalboard-3676](https://github.com/mattermost/focalboard/issues/3676).

Reviewer: ```suggestion
 - On Boards, selecting the **+ New** button below a column on the Kanban view doesn't always create a new card. As a workaround, set a new default card template by going to the dropdown menu from the blue **New** button on the header of the board, opening the **Options** menu on any card template, and selecting **Set as default** [issue-focalboard-3676](https://github.com/mattermost/focalboard/issues/3676).
```
